quantum/rgblight.h

```diff
@@ -267,6 +267,7 @@ void rgblight_timer_toggle(void);
 // if timeout is set, enable the rgblight idle feature
 #ifdef RGBLIGHT_IDLE_TIMEOUT
   #define RGBLIGHT_IDLE_ENABLE
+  #define RGBLIGHT_IDLE_MULTIPLIER 60000
```

Comment:
Maybe something like RGB_IDLE_MULTIPLIER_MSECS

Reply:
How about `RGB_DISABLE_AFTER_TIMEOUT`, as we're using that with RGB Matrix:

https://docs.qmk.fm/#/feature_rgb_matrix?id=additional-configh-options

Reply:
I'm ok with that, is just like to know what the unit of measurement is

Reply:
I'm ... not exactly sure, but the code is called: 
https://github.com/qmk/qmk_firmware/blob/cd7bc0313486b8786e52ee202834e4dc6f827a6b/quantum/rgb_matrix.c#L373

Reply:
@XScorpion2 what is the unit in, for `RGB_DISABLE_AFTER_TIMEOUT` (my brain is failing me)

Reply:
Technically it's 1.2s units. No idea why it was coded like that. Never bothered changing it so it made sense.

Reply:
As far as suggests on this PR goes, kill all the extra defines, just make this feature simple with a single define that is based on seconds. This PR is introducing 5 new defines for something that can be easily handled by 1.

Reply:
This should be changed to seconds or minutes then.

Reply:
This should be changed to seconds or minutes. Documentation says "ticks" which doesn't really mean anything

Reply:
I just submitted a pr for RGB Matrix to clean up it's idle timeout and change it to seconds: https://github.com/qmk/qmk_firmware/pull/6480